java/arcs/sdk/wasm/WasmJsInterop.kt

```diff
@@ -250,5 +250,106 @@ fun log(msg: String) {
     flush()
 }
 
+/** Get a combined handle to call events on. */
 fun <T, U> combine(handle1: WasmHandleEvents<T>, handle2: WasmHandleEvents<U>):
     WasmTupleHandle<T, U> = WasmTupleHandle(handle1, handle2)
+
+/** Get a combined handle to call events on. */
```

Comment:
Here, and for all the other methods below, you could just link to the first method (e.g. using `@see`) so that you don't have to copy and paste everything.